### PR TITLE
file limitation rules

### PR DESCRIPTION
**Pull request by williballenthin**
https://github.com/fireeye/capa/pull/591

### CODE REVIEW

**Comment by williballenthin** [comment]
merge #399 first to cleanup the diff with function-name feature

File: internal/limitation/file/internal-autoit-file-limitation.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      capa cannot handle AutoIt scripts. This means that the results will be misleading or incomplete.
+      You may have to analyze the file manually, using a tool like the AutoIt decompiler MyAut2Exe.
+    namespace: internal/limitation/file
+    author: william.ballenthin@fireeye.com
+    scope: file
+    examples:
+      - 55D77AB16377A8A314982F723FCC6FAE
+  features:
+    - or:
+      - match: compiler/autoit
```
Comment (mr-tz) [inline internal/limitation/file/internal-autoit-file-limitation.yml:20]: linter will require a new line at the end

**Review by mr-tz — CHANGES_REQUESTED** [review]
nice, see question on `internal` vs. using `capa` in other PR

also linter fails:
```
KeyError: 'anti-analysis/packer'
Error: Process completed with exit code 1.
```

**Comment by mr-tz** [comment]
VMProtect for https://github.com/fireeye/capa/issues/588

Is it worth to add this? Maybe it makes sense to display the limitation details, e.g. what lower rule names caused the limitation?

```yaml
rule:
  meta:
    name: (internal) VMProtect file limitation
    description: |
      This sample appears to be packed with VMProtect.
      
      Packed samples have often been obfuscated to hide their logic.
      capa cannot handle obfuscation well. This means the results may be misleading or incomplete.
      If possible, you should try to unpack this input file before analyzing it with capa.
    namespace: internal/limitation/file
    author: william.ballenthin@fireeye.com
    scope: file
    examples:
      - CD2CBA9E6313E8DF2C1273593E649682
  features:
    - or:
      - match: anti-analysis/packer/vmprotect

```

File: internal/limitation/file/internal-autoit-file-limitation.yml (hunk 1 above)
Comment (williballenthin) [reply internal/limitation/file/internal-autoit-file-limitation.yml:20]: linter bug #601

Comment (williballenthin) [reply internal/limitation/file/internal-autoit-file-limitation.yml:20]: fixed in https://github.com/fireeye/capa/commit/2706a7171e7dcd7dec689229d2ebe1bf4e735bbb

**Pull request merged**